src/Includes/Actions.php

```diff
@@ -61,4 +62,57 @@ public function register_assets() {
 			wp_add_inline_script( 'plausible-analytics', 'plausible("404",{ props: { path: document.location.pathname } });' );
 		}
 	}
+
+	/**
+	 * Create admin bar nodes.
+	 *
+	 * @since  1.2.0
+	 * @access public
+	 *
+	 * @return void
+	 */
+
+	public function admin_bar_node( $admin_bar ) {
+		// Add main admin bar node
+		$args = [
+			'id'    => 'plausible-admin-bar',
+			'title' => 'Plausible Analytics',
+		];
+		$admin_bar->add_node( $args );
+
+		// Add sub menu items
+
+		// Add link to view all stats
+		$args   = [];
+		$args[] = [
+			'id'     => 'view-analytics',
+			'title'  => 'View Analytics',
```

Comment:
Use `esc_html__( '<title>', '<text-domain>' ) for every such cases where title or labels are used